docs/self-managed/setup/deploy/gcp/google-gke.md

```diff
@@ -11,7 +11,9 @@ Camunda 8 Self-Managed can be deployed on any Kubernetes cluster using [Helm cha
 
 ### Google Cloud load balancer
 
-As Camunda Identity management endpoints like health endpoint don't use port 80, you could need a [custom health check configuration](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#direct_health) when using Google Cloud load balancer.
+Camunda Identity management endpoints, such as the health check endpoint, do not run on port 80. As a result, when using a Google Cloud Load Balancer, you may need a [custom health check configuration](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#direct_health).
```

Comment:
Do we mean port 8080?

Reply:
Actually, there are 2 different cases (Service port and Pod port, depending on the loadbalancer type).
I will add the 2 cases to avoid confusion.